documentation/asciidoc/computers/camera/synchronous_cameras.adoc

```diff
@@ -0,0 +1,83 @@
+== Synchronous Captures
+=== IMX477
+==== Preparation
+
+Connect a potential divider of 2x 1.5KOhm Resistors to 3v3 and ground, creating 1.65V. This can be connected to either Pi.
+
+Solder the GND and XVS touchpoints of each HQ Camera board to each other.
+
+Connect the XVS wires to the 1.65V potential divider pull up.
+
+==== Boot up both Raspberry Pis
+
+The file `/sys/module/imx477/parameters/trigger_mode` determines which board outputs pulses, or waits to recieve pulses (source and sink).
+This parameter can only be altered in superuser mode.
+
+On the sink, run:
+[,bash]
+----
+sudo su
+echo 2 > /sys/module/imx477/parameters/trigger_mode
+exit
+----
+
+On the source, run:
+[,bash]
+----
+sudo su
+echo 1 > /sys/module/imx477/parameters/trigger_mode
+exit
+----
+
+Start the sink running:
+[,bash]
+----
+libcamera-vid --frames 300 --qt-preview -o sync.h264
+----
+
+Start the source running
+[,bash]
+----
+libcamera-vid --frames 300 --qt-preview -o sync.h264
+----
+
+Frames should be synchronous. Use --frames to ensure the same amount of frames are captured, and that the recordings are exactly the same length.
+Running the sink first ensures that no frames are missed.
+
+==== Notes
+
+The potential divider is needed to pull up the XVS pin to high whilst the source is in an idle state.
+This ensures that no frames are created or lost upon startup. The source whilst initialising goes from LOW to HIGH which can trigger a false frame.
+
+=== IMX296
+==== Preparation
+
+Create a potential divider made of 2x 1.5KOhm Resistors to 3v3 and ground, creating 1.65V. This can be connected to either pi.
+
+Solder 2 wires to the XVS touchpoint on each board and connect both of these wires together to the 1.65V potential divider.
+
+Solder the GND of each Camera board to each other. Also solder 2 wires to the XHS touchpoints on each board and connect these. No pullup is needed for XHS pin.
+
+On the boards that you wish to act as sinks, solder the two halves of the MAS pad together. This enters the sensor into sink mode.
+
+==== Boot up both Raspberry Pis
+
+Start the sink running:
+[,bash]
+----
+libcamera-vid --frames 300 -o sync.h264
+----
+Allow a delay before you start the source running (see note below). Needs to be roughly > 2 seconds.
+
+Start the source running:
+[,bash]
+----
+libcamera-vid --frames 299 -o sync.h264
+----
+==== Note
+Due to limitations of the IMX296 sensor, we are unable to get the sink to record exactly the same amount of frames as the source.
+**The source will record one extra frame before the sink starts recording.** This will need to be accounted for later in the application.
```

Comment:
Is there some kind of `ffmpeg` command that can later (post-recording) be used to strip this extra frame?